app/router.js

```diff
@@ -46,6 +46,10 @@ router.map(function() {
       });
       this.route('export');
       this.route('sessions');
+      this.route('tickets');
+      this.route('tickets', function() {
+        this.route('ticket-details');
```

Comment:
Use the index sub-route instead to display the details only at `/events/<event_id>/tickets`